src/TwigComponent/src/Command/ComponentDebugCommand.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+use Symfony\UX\TwigComponent\ComponentFactory;
+use Symfony\UX\TwigComponent\Twig\PropsNode;
+use Twig\Environment;
+
+#[AsCommand(name: 'debug:component', description: 'Display current components and them usages for an application')]
+class ComponentDebugCommand extends Command
+{
+    public function __construct(private string $twigTemplatesPath, private ComponentFactory $componentFactory, private Environment $twigEnvironment, private iterable $components)
+    {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('name', InputArgument::OPTIONAL, 'A component name'),
+            ])
+            ->setHelp(<<<'EOF'
+                The <info>%command.name%</info> display current components and them usages for an application:
+
+                  <info>php %command.full_name%</info>
+
+                EOF
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $name = $input->getArgument('name');
+
+        if ($name) {
+            try {
+                $metadata = $this->componentFactory->metadataFor($name);
+            } catch (\Exception $e) {
+                $io->error($e->getMessage());
+
+                return Command::FAILURE;
+            }
+
+            $class = $metadata->get('class');
+            $allProperties = [];
+
+            if ($class) {
+                $propertyLabel = 'Properties (type / name / default value if exist)';
+
+                $reflectionClass = new \ReflectionClass($class);
+                $properties = $reflectionClass->getProperties();
+
+                foreach ($properties as $property) {
+                    if ($property->isPublic()) {
+                        $visibility = $property->getType()?->getName();
+                        $propertyName = $property->getName();
+                        $value = $property->getDefaultValue();
+
+                        $allProperties = [
+                            ...$allProperties,
```

Comment:
Funny to see that in PHP, I expected something more like:
```suggestion
                          $allProperties[] = $visibility.' $'.$propertyName.(null !== $value ? ' = '.$value : '')
```